CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+## Reporting Issues
+
+If you have found what you think is a bug, please [file an issue](https://github.com/pmndrs/jotai/issues/new).
+
+## Suggesting new features
+
+If you are here to suggest a feature, first create an issue if it does not already exist. From there, we will discuss use-cases for the feature and then finally discuss how it could be implemented.
+
+## Development
+
+If you have been assigned to fix an issue or develop a new feature, please follow these steps to get started:
+
+- Fork this repository
+- Install dependencies by running `$ yarn`
+- Link `jotai` locally by running `$ yarn link`
+- Auto-build files as you edit by running `$ yarn start`
+- Implement your changes and tests to files in the `src/` directory and corresponding test files
+- To run examples, follow their individual directions. Usually this is just `$ yarn && yarn start`.
+- To run examples using your local build, link to the local `jotai` by running `$ yarn link jotai` from the example's directory
+- Git stage your required changes and commit (see below commit guidelines)
+- Submit PR for review
+
+### Type
+
+Must be one of the following:
+
+- **feat**: A new feature
+- **fix**: A bug fix
+- **docs**: Documentation only changes
+- **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing
+  semi-colons, etc)
```

Comment:
```suggestion
```

I'd include it in **refactor**. (or **chore** if it's like changing the prettier version/config.)